File: rollon_erpnext/rollon_erpnext/report/cr_stock_ageing/cr_stock_ageing.py
# Copyright (c) 2013, Biradar Technologies and contributors
# For license information, please see license.txt

from __future__ import unicode_literals
import frappe
from frappe import _
from frappe.utils import flt, getdate

def execute(filters=None):
	columns, data = [], []
        columns=get_columns()
        data=get_data(filters)
	return columns, data

def get_columns():
    return [
    _("Item Name") + ":Link/Item:300",
    _("Stock Balance") + ":Float:120"
    ]

def get_data(filters):
   if filters.get("item_group"):
    return frappe.db.sql ("""
    SELECT
    A.item_name,
    A.item_name
   
    FROM 
    `tabItem` AS A
    
    Where
    A.item_group='%s'

    ORDER BY A.item_name ASC """ %("item_group"), as_list=1)

File: rollon_erpnext/rollon_erpnext/report/cr_stock_ageing/cr_stock_ageing.js
// Copyright (c) 2016, Biradar Technologies and contributors
// For license information, please see license.txt
/* eslint-disable */

frappe.query_reports["CR Stock Ageing"] = {
	"filters": [
        {
        "fieldname":"range",
        "label": ("Range"),
        "fieldtype":"Select",
        "options":["0-30","31-60","61-90","91-180","181-Above"],
        "default":"0-30"
        },

        {
        "fieldname":"item_group",
        "label":("Item Group"),
        "fieldtype":"Link",
        "options":"Item Group",
        "default":"Raw Material"
        }


    ]

}
